index.css

```diff
@@ -0,0 +1,68 @@
+body{
+  margin: 0;
+}
+/*start page css*/
+.page{
+  width: 495px;
+  margin: auto;
+  text-align: center;
+}
+.page .header{
+  height: 50px;
+  border: 1px solid red;
+  padding: 2px 0 2px 20px;
+}
+.page .content{
+  border: 1px solid red;
+  min-height: 500px;
+}
+.page .footer{
+  height: 50px;
+  border: 1px solid red;
+}
+/*end page css*/
+
+/*start header css*/
+.header .header-left{
```

Comment:
we should create 2 columns component for header-left and header-right. this was stand alone component